packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1525,6 +1525,25 @@ mixin WidgetInspectorService {
     return false;
   }
 
+  /// Notify debuggers to open an inspector on the object.
+  void _sendInspectEvent(Object? object){
+    inspect(object);
+
+    final _Location? location = WidgetInspectorService.instance._getSelectedSummaryWidgetLocation(null);
```

Comment:
can this just be `_getSelectedSummaryWidgetLocation` instead of `WidgetInspectorService.instance._getSelectedSummaryWidgetLocation`?

Reply:
oops yes ty :)